### PR TITLE
User story 11

### PR DESCRIPTION
**Pull request by MatthewTLim**
# Describe the changes below:
This PR will add all tests and user functionality for user story 11
# Relevant user story:
user story 11
# Specific feedback requests? Include file name and line:

# Before submitting, check the following:
- [X] Entire test suite is passing
- [99.27%] SimpleCov test percentage

# Note any co-authors / anything else you'd like to add:

### CODE REVIEW

File: app/controllers/items_controller.rb [hunk 1]
```diff
@@ -29,11 +28,27 @@ def update
...
+    @merchant = Merchant.find(params[:merchant_id])
+  end
 
-  def item_params
-    params.require(:item).permit(:name, :description, :unit_price, :id, :merchant_id, :status)
+  def create
+    @merchant = Merchant.find(params[:merchant_id])
+    @item = @merchant.items.new(new_items_params)
+    if @item.save
+      redirect_to merchant_items_path(@merchant)
```
Comment (dani-wilson) [inline app/controllers/items_controller.rb:39]: Nice!

File: app/views/items/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<%= form_with(
+  model: @item,
```
Comment (dani-wilson) [inline app/views/items/new.html.erb:2]: I spy a shiny new technique. Nice!

**Review by dani-wilson — APPROVED** [review]

File: app/controllers/items_controller.rb [hunk 3]
```diff
@@ -29,11 +28,27 @@ def update
...
 
+  private
+
+  def new_items_params
+    params.permit(:name, :description, :unit_price, :status)
+  end
 
+  def item_params
+    params.require(:item).permit(:name, :description, :unit_price, :status)
+  end
```
Comment (westonio) [inline app/controllers/items_controller.rb:53]: Nice use of strong params and the new .require statement 🙌 Looks amazing

File: app/views/items/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  local: true,
+  data: { turbo: false },
+  url: merchant_items_path(@merchant)
+  )do |f| %>
```
Comment (westonio) [inline app/views/items/new.html.erb:6]: Heads up - looks like an accidental `)` on line 6

**Review by westonio — APPROVED** [review]
Looks great, aside from a potentially erroneous `)` as mentioned in a previous comment.

**Pull request merged**